nebula.proto

```diff
@@ -63,3 +63,16 @@ message NebulaHandshakeDetails {
   uint64 Time = 5;
 }
 
+message NebulaControl {
+  enum MessageType {
+    None = 0;
```

Comment:
Two things: first, it was inspired by the code above:
```
message NebulaMeta {
  enum MessageType {
    None = 0;
    HostQuery = 1;
    HostQueryReply = 2;
```

When in doubt, cut & paste.

Secondly, Go defaults unset int values to 0. Reserving the 0 value as 'none' helps ensure that developers specify the Type value explicitly when constructing Control messages.

Without reserving 0 (and setting CreateRelayRequest to enum value 0), a CreateRelayRequest could look like this:
```
			req := NebulaControl{
				Type:                NebulaControl_CreateRelayRequest,
				InitiatorRelayIndex: index,
				RelayFromIp:         uint32(h.vpnIp),
				RelayToIp:           uint32(target),
			}
```
or this (missing the Type value being set):
```
			// Send a CreateRelayRequest to the peer.
			req := NebulaControl{
				InitiatorRelayIndex: index,
				RelayFromIp:         uint32(h.vpnIp),
				RelayToIp:           uint32(target),
			}
```

Reserving '0' for None makes the second construction more detectable (because the peer will reject it.)